### PR TITLE
Payment stripe

### PR DESCRIPTION
**Pull request by mwangiii**
# Fixes Issue/Linear Ticket

> This could be an existing issue or a linear ticket
>
> - Github Issue Example: My PR Closes #{ISSUE}
> - Linear Ticket Example: Fixes ID-#{ISSUE}

## Changes proposed

> Talk about the things you did eg. files changes, dependencies installed e.t.c

## Check List (Check all the applicable boxes)

:rotating_light:Please review the [style guide for contributing](add link here) and [guidelines for contributing](add link here) to this repository.

- [ ] My code follows the code style of this project.
- [ ] This PR does not contain plagiarized content.
- [ ] The title and description of the PR is clear and explains the approach.
- [ ] I am making a pull request against the **main branch** (left side).
- [ ] My commit messages styles matches our requested structure.
- [ ] My code additions will fail neither code linting checks nor unit test.
- [ ] I am only making changes to files I was requested to.

### CODE REVIEW

File: server-script/logs/devoutput.log
Comment (incredible-phoenix246) [inline server-script/logs/devoutput.log:1]: you are not allowed to touch this file brother

**Review by incredible-phoenix246 — CHANGES_REQUESTED** [review]

File: server-script/logs/devoutput.log
Comment (mwangiii) [reply server-script/logs/devoutput.log:1]: Sorry

File: src/config/index.ts
Comment (incredible-phoenix246) [inline src/config/index.ts:1]: why import express in a config file

**Review by incredible-phoenix246 — CHANGES_REQUESTED** [review]

File: src/config/index.ts
Comment (mwangiii) [reply src/config/index.ts:1]: image
I think someone made changes to get past the stripe api key error

**Pull request closed without merging**